### PR TITLE
Add columns to collector model with migration and tests

### PR DESCRIPTION
**Pull request by pallavi2209**
Only migration and models changes with tests. Will do another PR for swagger and API changes.

### CODE REVIEW

File: migrations/20170911221051-add-osInfo-processInfo-version-to-collector.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        type: Sequelize.JSONB,
+        allowNull: true,
+      })
+      .then(() => qi.addColumn(TBL, 'processInfo', {
+        type: Sequelize.JSONB,
+        allowNull: true,
+      }))
+      .then(() => qi.addColumn(TBL, 'version', {
+        type: Sequelize.STRING,
+        allowNull: false,
```
Comment (iamigo) [inline migrations/20170911221051-add-osInfo-processInfo-version-to-collector.js]: does migration work if adding a column with `allowNull: false` if there are already records in the db? might have to provide default value (if so, maybe just set default value to `0.0.0` to imply we don't have a version yet)?

Comment (shriramshankar) [reply migrations/20170911221051-add-osInfo-processInfo-version-to-collector.js]: We had the default version set as 1.0.0 for SGT. Wondering if we want to do the same for the collectors too.?

Comment (iamigo) [reply migrations/20170911221051-add-osInfo-processInfo-version-to-collector.js]: default ver for SGT is different ... here we're just making sure nothing breaks if a record existed before this column was added

**Comment by pallavi2209** [comment]
@iamigo @shriramshankar updated the PR

**Pull request merged**